src/main/java/Main.java

```diff
@@ -0,0 +1,51 @@
+import dataStructures.InvertedIndex;
+import filter.WordValidator;
+import filter.tokenizer.NGramTokenizer;
+import filter.normalizer.UpperCaseNormalizer;
+import filter.tokenizer.SplitTokenizer;
+import reader.TXTReader;
+import search.searchMode.Search;
+import search.searchMode.advancedSearch.AdvancedSearch;
+
+import java.io.*;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.util.*;
+
+public class Main {
+
+    public static final String path = "./textFiles";
+    static long startTime, endTime;
+
+    public static void main(String[] args) throws IOException {
+        startTime = System.currentTimeMillis();
+        InvertedIndex invertedIndex = InvertedIndex.builder().
+                tokenizer(new SplitTokenizer("[^\\da-zA-Z]+")).
+                normalizer(new UpperCaseNormalizer()).
+                doStem(false).
+                wordValidator(new WordValidator(true)).
+                build();
+        DataManager dataManager = new DataManager(invertedIndex);
+        dataManager.createDatabase(new TXTReader(path));
+        String query = getQuery();
+        Search search = new AdvancedSearch(invertedIndex, query);
+        printDocuments(search.sortResult(search.getAllDocuments()));
```

Comment:
It's better to have no input parameter for sortResult and it should sort the documents in private field inside search object.